core/domain/email_manager.py

```diff
@@ -224,6 +224,9 @@ def _send_email(
             the email.
         reply_to_id: str or None. The unique reply-to id used in reply-to email
             address sent to recipient.
+
+    Returns:
+        None.
```

Comment:
FYI, I just noticed that this function doesn't actually need to return the result of run_in_transaction() on line 269. I think we can drop the return (and modify other stuff accordingly), if it helps.

This might be applicable for some other functions too.

Reply:
So here can we remove all `return` statements from lines 244, 256 and 269?

Reply:
That breaks the logic. So, no.

(Make sure you read the functions carefully and understand what they're doing. The fixes needed to bring the codebase into compliance with the linter are sweeping and should never be done cookie-cutter. There is a chance of breaking things here.)

Reply:
Got it. Let me look into the "None" issue.